scripts/run_linux.sh

```diff
@@ -95,11 +95,27 @@ function install_dependencies() {
         local kotaemon_root="$(pwd)/libs/kotaemon"
         local ktem_root="$(pwd)/libs/ktem/"
 
-        echo "" && echo "Install kotaemon's requirements"
-        python -m pip install -e "$kotaemon_root"
+        if [ -f "$(pwd)/VERSION"]; then
```

Comment:
You are missing a space before the closing bracket, resulting in the `VERSION` file not being found and it will always install the latest version.

Reply:
nice catch

Reply:
fixed